mvi-core/src/androidMain/AndroidManifest.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="net.humans.kmm.mvi" />
\ No newline at end of file
+<manifest />
```

Comment:
Why? not related after `namespace` inclusion?

Reply:
manifest attr `package` is replaced by `namespace` property in gradle config for android modules.  
not required anymore.